fhir-path/src/test/java/com/ibm/fhir/path/test/MemberOfFunctionTest.java

```diff
@@ -118,7 +118,7 @@ public void testMemberOfFunction6() throws Exception {
         Assert.assertEquals(result, SINGLETON_FALSE);
     }
 
-    @Test
+    @Test(enabled = false)
```

Comment:
why is it disabled?  Maybe add a comment to explain the why?

Reply:
if its because "`in` just doesn't work like this any more" then maybe it should just be deleted?

Reply:
I'll add the comment in my next PR.